### DISCUSSION TITLE
Aderyn tool is not working properly on WINDOWS-11

### DISCUSSION THREAD

**Discussion by gurwindersinghdev** [Cyfrin/security-and-auditing-full-course-s23#48] [Q&A]
![ad](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/112465044/9a0138e7-4142-404d-ab8f-19125bce4d0e)

``` aderyn . ```            (command is working fine)

this tool doesn't give me reports in Details ? like it should work as it shows in the video walkthrough 

how do i fix this error

Tools - Vs code, 
terminal - gitbash

**Comment by TilakMaddy** [comment]
Could you try `cargo install --force aderyn` and get back ?

**Reply by hamzalagrari** [reply]
hello bro i have a question :
i follewed all the steps installing aderyn but i still get this error every time i run "aderyn ."
error:"aderyn: command not found" 
do you have a solution for that ?

**Reply by TilakMaddy** [reply]
Are you using WSL ?

**Reply by hamzalagrari** [reply]
yes

**Reply by TilakMaddy** [reply]
you have to then find aderyn binary locally (see inside $HOME/.cargo ) then add it to $PATH

**Reply by hamzalagrari** [reply]
how i can find aderyn binary ? is there a command or should i look for it manually ?

**Reply by hamzalagrari** [reply]
thx bro i figured that out  and it worked fine now

**Reply by TilakMaddy** [reply]
You are welcome : ) 🤗

**Reply by Yahya-Alshamy** [reply]
@hamzalagrari 
I'm having the same problem, but adding the bin to $PATH didn't fix it. Could you please let me know if you tried any other solutions to get it fixed?

**Comment by cromewar** [comment]
Please @PatrickAlphaC a hand on this one.

**Reply by ChinHaoTsou** [reply]
I got the same issue in windows 11, no details in report. It seems only happened in windows 11?

**Reply by TilakMaddy** [reply]
@ChinHaoTsou  please tell me what command you have executed and in which directory? (is there a foundry.toml / hardhat.config.js/ts)

**Reply by TilakMaddy** [reply]
This person also uses WSL and he seems to have gotten it. So maybe not a windows issue
https://github.com/Cyfrin/security-and-auditing-full-course-s23/discussions/61

But I don't know...will look into it

**Reply by ChinHaoTsou** [reply]
<img width="334" alt="image" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/44434570/be72a031-4450-4280-88dc-77d197aeb7c8">
I ran aderyn . in git bash, and it seems working fine but report without any details.

**Comment by gurwindersinghdev** [comment]
![Code_yjsKHuJL8X](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/112465044/f5b2484d-9129-41e3-8530-dcc6f1efa7a0)

aderyn .
Detecting framework...
Error detecting framework

**Reply by TilakMaddy** [reply]
Run  command `ls` and show output

**Comment by EngrPips** [comment]
@gurwindersinghdev  i assume you are good with aderyn now
